index.html

```diff
@@ -5,25 +5,61 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="style.css" />
+    <script
+      src="https://kit.fontawesome.com/e98346ef52.js"
+      crossorigin="anonymous"
+    ></script>
     <title>Quiz App - by Alice</title>
   </head>
   <body>
     <header><h1>Quiz App</h1></header>
     <main>
-      <!-- Platzhalter für question cards -->
       <section>
-        <h2>QUESTION:</h2>
-        <!-- Java Script für output von create.html fehlt noch-->
-        <p>Hier steht eine Frage</p>
-        <button>Show Answer/Hide Answer</button>
-        <ul>
-          <li>Tag</li>
-          <li>Tag</li>
-          <li>Tag</li>
-          <li>Tag</li>
-        </ul>
+        <article class="question__card">
+          <h2 class="question__card--title">Question:</h2>
+          <!-- Java Script für output von create.html fehlt noch-->
+          <p class="question__card--question">
+            Hier steht eine Frage<br />
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolore hic
+            possimus eaque nam in perspiciatis laudantium doloribus enim.
+          </p>
+          <button class="question__card--button">
+            Show Answer
+            <!-- /Hide Answer -->
+          </button>
+          <div class="question__card--all-tags">
+            <div class="question__card--tag">Tag</div>
+            <div class="question__card--tag">Tag</div>
+            <div class="question__card--tag">Tag</div>
+            <div class="question__card--tag">Tag</div>
+          </div>
+          <a class="question__card--bookmark" href="bookmark.html" alt="Home">
+            <i class="fa-solid fa-bookmark"></i>
+          </a>
```

Comment:
This one is more of a question: I'm not sure if it is user-friendly that every time you click on a bookmark will guide the user to the bookmark page. I would suggest you to work with a button and let the user decide for himself or herself when he or she wants to see the bookmark page. If you want to work with a button, then the value of the `alt` tag should not be `Home`.